lib/uploadcare/configuration.rb

```diff
@@ -0,0 +1,9 @@
+require 'singleton'
+
+module Uploadcare
+  class Configuration
```

Comment:
Should we move this class to the gem?

Reply:
Same here. I think that this is framework specific and this is a reason not to place it to the ruby-gem.